src/main/java/com/extremelyd1/listener/ChatListener.java

```diff
@@ -32,10 +33,16 @@ public void onAsyncPlayerChat(AsyncPlayerChatEvent e) {
         Player player = e.getPlayer();
 
         Team team = game.getTeamManager().getTeamByPlayer(player);
+        ChatChannelController.ChatChannel chatChannel = game.getChatChannelController().getPlayerChatChannel(player);
         if (team == null) {
             Bukkit.broadcastMessage(
                     player.getName() + ": " + e.getMessage()
             );
+        }
+        else if (chatChannel == ChatChannelController.ChatChannel.GLOBAL) {
```

Comment:
The else-if-statement can be placed after the closing curly brace of the previous line.

Reply:
done